packages/one-app-runner/bin/one-app-runner.js

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const pkgUp = require('pkg-up');
+const startApp = require('../src/startApp');
+
+const createYargsConfig = () => {
+  const packageJsonPath = pkgUp.sync();
+  // eslint-disable-next-line global-require, import/no-dynamic-require
+  const packageJson = require(packageJsonPath);
+
+  // dont agree with this rule in this instance. Rewriting this code would over-complicate it
+  // eslint-disable-next-line no-mixed-operators
+  const oneAppRunnerConfig = packageJson['one-amex'] && packageJson['one-amex'].runner || {};
```

Comment:
did you try this for the rule ?
```suggestion
  const oneAppRunnerConfig = (packageJson['one-amex'] && packageJson['one-amex'].runner) || {};
```

Reply:
Fixed in last commit.